### DISCUSSION TITLE
<My repository does not appear in the browser results>

### DISCUSSION THREAD

**Discussion by BraisCabo** [community/community#45060] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

If I search in the browser my name or the name of the repo it doesnt appear in search results. How can I fix that?

**Comment by SilasRodrigues19** [comment]
Hey @BraisCabo 

Write a good description for your repository and make a very detailed README file, this will help a lot.

If you want, you can take a look at the README of my repos to get an idea

**Comment by BraisCabo** [comment]
Thanks @SilasRodrigues19 

It also happens with my proffile, if I search it on google i dont get any results. Can i solve that?

**Reply by SilasRodrigues19** [reply]
@BraisCabo 

Mine after I improved the description and README, it started to be displayed by Google

**Reply by SilasRodrigues19** [reply]
For example, searching my repository called Cities: https://www.google.com/search?q=silasrodrigues19+Cities&oq=silasrodrigues19+cities&aqs=chrome.0.69i59j69i57j69i60l3.7560j0j7&sourceid=chrome&ie=UTF-8

**Comment by tanner-miller5** [comment]
If you are using an organization you need to click on your organizations (top right of window under) and then click the specific organization (main pane window). Then you should see the repos in the organization and search. Another possibility is you're on the wrong account or looking at your organizations repos instead of your personal repos.
